pkg/executables/kubectl.go

```diff
@@ -1205,6 +1215,26 @@ func (k *Kubectl) SearchVsphereMachineConfig(ctx context.Context, name string, k
 	return response.Items, nil
 }
 
+// SearchTinkerbellMachineConfig returns the list of TinkerbellMachineConfig in the cluster.
+func (k *Kubectl) SearchTinkerbellMachineConfig(ctx context.Context, name string, kubeconfigFile string, namespace string) ([]*v1alpha1.TinkerbellMachineConfig, error) {
```

Comment:
Are we searching? This looks like standard retrieval, i.e. `Get*` or `Retrieve*`.

Reply:
I wanted to keep the name in sync with other providers. https://github.com/aws/eks-anywhere/blob/main/pkg/executables/kubectl.go#L1189
I am fine either ways though.